### PR TITLE
Feature/us6284 list view

### PR DESCRIPTION
**Pull request by eymiha**
Show a list of results when clicking "List" on the search bar. Within a stone's throw of "fine-tuning" to the latest Ample renditions. Click on a gathering, go to host details. Click on a person, say hi. Click on a church, nada.

Also, working ahead because we were right in the affected code, proximities and participant counts _are_ completed in this feature.

### CODE REVIEW

File: src/app/components/list-entry/list-entry.component.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+export class ListEntryComponent {
+  @Input() firstName: string = " ";
+  @Input() lastName: string = " ";
+  @Input() siteName: string = "";
+  @Input() type: number;
+  @Input() proximity: number = 0;
+  @Input() description: string = "";
+  @Input() address: Address = null;
+  @Input() participantId: number = 0;
+  @Input() participantCount: number = 0;
```
Comment (vchtelmakh) [inline src/app/components/list-entry/list-entry.component.ts]: Would this be better as a ListEntry model with all these properties on it?

File: src/app/components/list-view/list-view.component.spec.ts [hunk 2]
```diff
@@ -59,7 +59,7 @@ describe('Component: List View', () => {
 
   });
 
-  it('should create an instance', () => {
+  fit('should create an instance', () => {
```
Comment (vchtelmakh) [inline src/app/components/list-view/list-view.component.spec.ts]: Unfocus tests please.

File: src/app/components/list-view/list-view.component.ts [hunk 3]
```diff
@@ -1,13 +1,35 @@
...
+import { Address } from '../../models/address';
+import { Pin, pinType } from '../../models/pin';
+import { PinSearchResultsDto } from '../../models/pin-search-results-dto';
+import { UserLocationService } from  '../../services/user-location.service';
 
 @Component({
   selector: 'app-listview',
   templateUrl: 'list-view.component.html'
 })
 export class ListViewComponent {
```
Comment (sseissiger) [inline src/app/components/list-view/list-view.component.ts]: to get additional checks please add implements OnInit

File: .env.dave-vm [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Adjust these environment variables
+CRDS_API_ENDPOINT=http://dave-vm:49380/
+CRDS_ENV=int
+CRDS_COOKIE_DOMAIN=localhost
+CRDS_STRIPE_PUBKEY=pk_test_U8U15gSZFM4AQtPDLHYnKWqH
+PARENT_WHITELIST="^https:\/\/.*\.crossroads\.net(?::\d{1,5})?|^http:\/\/localhost(?::\d{1,5})?"
```
Comment (vchtelmakh) [inline .env.dave-vm]: Remove this file.

**Pull request merged**